frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/hooks/useSettingsModalMenuTabConfigs.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import type { SettingsModalTabId } from '../../../../../../types/SettingsModalTabId';
+import {
+  InformationSquareIcon,
+  SidebarBothIcon,
+  ShieldLockIcon,
+  TimerStartIcon,
+} from '@studio/icons';
+import { useTranslation } from 'react-i18next';
+import type { StudioContentMenuButtonTabProps } from '@studio/components';
+
+const aboutTabId: SettingsModalTabId = 'about';
+const setupTabId: SettingsModalTabId = 'setup';
+const policyTabId: SettingsModalTabId = 'policy';
+const accessControlTabId: SettingsModalTabId = 'access_control';
+
+export const useSettingsModalMenuTabConfigs = () => {
+  const { t } = useTranslation();
+
+  const getMenuTabConfigs = (): StudioContentMenuButtonTabProps<SettingsModalTabId>[] => {
+    return [
```

Comment:
Kan ikke dette returneres direkte fra hooken?

Reply:
Jo, kan det! Returnerte som funksjon siden det er vanlig for hooks, men er ikke nødvendig i dette tilfellet